portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,14 @@
     <script src="script.js"></script>
     <link href="https://fonts.googleapis.com/css2?family=Quicksand&family=Nunito&display=swap" rel="stylesheet">
     <script src="https://kit.fontawesome.com/906fde76e4.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBtUOgoOPElbga5Y4oh0xLYKsbthvAds4Q"></script>
   </head>
-  <body onload="displayServerContent()">
+  <body onload="createMap();displayServerContent()">
```

Comment:
It's probably better to only have one statement in your html binding. Can you create a new function that you can call from here that both creates a map and displays server content?